### DISCUSSION TITLE
installing IRAF 2.17 on macOS monterey

### DISCUSSION THREAD

**Discussion by BOOM5s** [iraf-community/iraf#307] [Q&A]
hello there,
I am trying to install IRAF 2.17 in my rooted directory.

[error IRAF.txt](https://github.com/iraf-community/iraf/files/11730420/error.IRAF.txt)

**Comment by olebole** [comment]
```
…
+ mv f2c ../../bin/f2c.e
mv: rename f2c to ../../bin/f2c.e: Not a directory
make: *** [sysgen] Error 1
```
The issue is probably the tame as in #304: in 2.17, you should do the following commands to build IRAF, as described in the [installation instructions](https://iraf-community.github.io/install.html) or in the local file `INSTALL.md`:
```
./install
make macos64
make sysgen
```

**Reply by BOOM5s** [reply]
thank you sir for the quick reply.
I have execute ./install after error 
the command line is show in file1.txt
after this I use make macos64 the result is showed in file 2.txt
make sysgen gives the same error as on where this discussion started file 3.txt
[File 1.txt](https://github.com/iraf-community/iraf/files/11736452/File.1.txt)
[file 2.txt](https://github.com/iraf-community/iraf/files/11736457/file.2.txt)
[File 3.txt](https://github.com/iraf-community/iraf/files/11736482/File.3.txt)

**Reply by olebole** [reply]
You should check that `unix/bin` actually is a link and points to an existing `bin.macos64` directory. If not, then the best would be to start from a fresh unpacked tarball. Note that you need to use the `tar.gz` file, not the `zip` file, as links are not properly preserved in zip ziles.

**Reply by BOOM5s** [reply]
ok thank you Sir

**Comment by zetorman** [comment]
Hello, I am using Monterey on M1 Pro (MB Pro 14 2021 model). I am trying to install IRAF using this [instruction]( https://iraf-community.github.io/install). I downloaded Xcode and IRAF tar package iraf-2.17.1.tar (there is no .gz at the end). But "tar zxf /<path>/iraf-2.17.1.tar.gz" is not working for me. So I tried "tar zxf /<path>/iraf-2.17.1.tar", also not working. That is why I started from this line "cd iraf-2.17.1/". When I do "make 2>&1 | tee build.log" and "make test", some errors appear. According to other topics, the problem is caused probably by not using this line "tar zxf /<path>/iraf-2.17.1.tar.gz". Or maybe by X11 (also not able to install)
But how can I start from "tar zxf /<path>/iraf-2.17.1.tar.gz" when it shows no such file or directory? I also attached the buildlog. Thank you. 
[buildlog.txt](https://github.com/iraf-community/iraf/files/12701774/buildlog.txt)
